demo/package.json

```diff
@@ -12,6 +12,8 @@
     "rlnjs": "^2.0.8"
   },
   "devDependencies": {
+    "@cryptkeeper/providers": "../dist/js/src/providers",
```

Comment:
As we agreed on the call, it's better to move it to a separate repository.
If it's not a fast solution, I'd keep types and provider inside demo because it's not possible to use without building our extension.